health/health_test.go

```diff
@@ -284,6 +306,24 @@ func initProvider(t *testing.T, providerType pt.Type, executorType pc.SparkExecu
 			SSLMode:  "disable",
 		}
 		return postgresConfig.Serialize()
+	case pt.ClickHouseOffline:
+		db := checkEnv("CLICKHOUSE_DATABASE")
+		user := checkEnv("CLICKHOUSE_USER")
+		password := checkEnv("CLICKHOUSE_PASSWORD")
+		host := getEnv("CLICKHOUSE_HOST", "localhost")
+		port := uint64(9000)
+		port, _ = strconv.ParseUint(getEnv("CLICKHOUSE_PORT", "9000"), 10, 16)
+		ssl := false
+		ssl, _ = strconv.ParseBool(getEnv("CLICKHOUSE_SSL", "false"))
```

Comment:
There is a [GetEnvBool](https://github.com/featureform/featureform/blob/main/helpers/env.go#L47-L52) that you can leverage